shared/src/commonMain/kotlin/org/hyperskill/app/paywall/view/PaywallViewStateMapper.kt

```diff
@@ -18,7 +20,9 @@ class PaywallViewStateMapper(
                     buyButtonText = resourceProvider.getString(
                         SharedResources.strings.paywall_mobile_only_buy_btn,
                         state.formattedPrice
-                    )
+                    ),
+                    isToolbarVisible = state.paywallTransitionSource != LOGIN,
```

Comment:
I think that this is strange that toolbar is only visible when `State.Content`, visibility should be configured on initial presentation.

JFYI: iOS will ignore `isToolbarVisible` property, because on iOS that means that should Paywall module be embedded in the navigation controller or not before presentation (before rendering).